modules/core/package.json

```diff
@@ -35,28 +35,25 @@
     "build-bundle": "ocular-bundle ./bundle.ts"
   },
   "dependencies": {
-    "@loaders.gl/core": "^3.0.0",
```

Comment:
I'd prefer to use the version of loaders on master since there's an issue with GIF encoding the videos module in later versions, and we're not using anything from the newer version of loaders

Reply:
`^3.4.13` ?

Reply:
In order to bump to deck9.1, I need `@loaders.gl/core@"^4.2.0"`, where is the encoding issue discussed?

Reply:
https://github.com/visgl/loaders.gl/issues/2164

Reply:
It actually still affects this branch too after removing the resolution